.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,46 @@
+<!--
+For high severity issues, please contact AWS Support directly.
```

Comment:
I might phrase this as "urgent operational issues" instead. I'd also like to see a link to AWS Support contact info. If somebody is in operational pain, they'll appreciate not having to dig for such details themselves.